modules/virt-installing-a-watchdog.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-a-watchdog.adoc
+
+[id="virt-defining-a-watchdog_{context}"]
+= Installing a watchdog device
+
+To activate a watchdog device, install the `watchdog` package on your virtual machine and start the watchdog service.
+
+.Procedure
+
+. Install the watchdog package and dependencies:
++
+[source,terminal]
+----
+$ yum install watchdog
+----
+
+. Uncomment the following line in the `/etc/watchdog.conf` file, and save the changes:
++
+[source,terminal]
+----
+$ watchdog-device = /dev/watchdog
```

Comment:
there's no dollar sign in this file
```suggestion
#watchdog-device = /dev/watchdog
```
but more importantly - is it really required? I would guess that `/dev/watchdog` is the default and should not be touched.

Reply:
I tested this today and it turns out that on RHEL8, updating the `/etc/watchdog.conf` file by uncommenting this entry is actually needed.
- Uncomment the following line was required,  #watchdog-device = /dev/watchdog
- Else Verification step with "kill -9 proccessid" failed.
- Also we see the below difference "with comment" and "without comment"
            "systemctl status watchdog" shows "alive=[none]" with comment
            "systemctl status watchdog" shows below without comment
                 watchdog[13737]:alive=/dev/watchdog"
                 watchdog[13737]: hardware watchdog identity: i6300ESB timer